### DISCUSSION TITLE
Whether import .env and moleculer runner with --env all just the same target for using environment variables?

### DISCUSSION THREAD

**Discussion by liaodalin19903** [moleculerjs/moleculer#1197] [Q&A] [answered]
We can use below script to load .env file:
```
# Load the default .env file from current directory
$ moleculer-runner --env
```

---

in my experience, we can use import  way to load the environment variables.
such as:
```
// db.mixin.js
require('dotenv').config();
```

whether the both methods' target all just provide environment variables to moleculer?

**Comment by icebob** [comment, marked as answer]
The `--env` does the same as you wrote `require('dotenv').config();`
